services/gateway-api/src/circuit_breaker/__init__.py

```
@@ -0,0 +1,99 @@
+    "RequestQueue",
+    "QueuedRequest",
+    "global_fallback_handler",
+
+    # Service-specific breakers
+    "OpenAICircuitBreaker",
+    "setup_openai_health_check",
+    "AnthropicCircuitBreaker",
+    "setup_anthropic_health_check",
+    "MetaCircuitBreaker",
+    "setup_meta_health_check",
+]
```

Ruff (0.14.8)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)